src/main/java/com/hwr/hwrzeiterfassung/database/controller/DayController.java

```diff
@@ -0,0 +1,18 @@
+package com.hwr.hwrzeiterfassung.database.controller;
+
+import com.hwr.hwrzeiterfassung.database.repositorys.DayRepository;
+import com.hwr.hwrzeiterfassung.database.tables.Day;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/day")
+public class DayController {
+    @Autowired
+    private DayRepository dayRepository;
+
+    public Day getDayById(long dayId){
+        return dayRepository.findById(dayId).get();
```

Comment:
is das nicht unnötig, da es eigentlich nur eine andere genauso lange Methode aufruft

Reply:
Neue Idee: Ich könnte da ein Troubleshooting wie in PR #51 einfügen und dafür meinen doch eher unpäzisen Try-Catch-Block im `TimeEntryController` rausnehmen